server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/wallets/bitgo/v2/dto/BitGoCoinRequest.java

```diff
@@ -21,11 +21,13 @@
     private String address;
     private Integer amount;
     private String walletPassphrase;
+    private String feeRate;
 
-    public BitGoCoinRequest(String address, Integer amount, String walletPassphrase) {
+    public BitGoCoinRequest(String address, Integer amount, String walletPassphrase, String feeRate) {
```

Comment:
feeRate should be Integer please change it here.

Reply:
The API allows to use either string or integer[1]. The reasoning for choosing String is: Since this parameter overrides the BitGo's fee calculation algorithm, an empty value is preferred, for when it is not set, rather than a default integer. Choosing a default fixed fee rate has its own complexity.

1- https://app.bitgo.com/docs/#operation/express.wallet.sendcoins

Reply:
According the API documentation the value should be satoshis/kilobyte, so Integer could create more problems than this parameter aims to solve. We are refactoring the implementation and will propose a new approach.